pkg/roachprod/roachprod.go

```diff
@@ -1471,20 +1464,14 @@ func destroyCluster(cld *cloud.Cloud, l *logger.Logger, clusterName string) erro
 		l.Printf("Destroying cluster %s with %d nodes", clusterName, len(c.VMs))
 	}
 
-	if err := deleteClusterConfig(clusterName, l); err != nil {
+	if err := promhelperclient.NewPromClient().DeleteClusterConfig(context.Background(),
```

Comment:
Should this be implemented down a layer in `cloud.DestroyCluster`? That's what's called by `gc.go` and would remove the need to call it there.

Reply:
I could not combine them as there seem to be different flows for destroy and gc.